src/style.css

```diff
@@ -1 +1,52 @@
-/* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.body {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 140px;
+  height: 160px;
+  padding: 22px;
```

Comment:
It's not recommended to fix container sizes unless it's a specific requirement (line 10 and line 11). Let the content size dictate the container's dimensions to avoid overflow or accidental scroll bars.